drafts/2019-10-22-liberty-dev-mode.adoc

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Open Liberty development mode"
+categories: blog
+author_picture: https://avatars2.githubusercontent.com/u/29640130
+author_github: https://github.com/ericglau
+seo-title: Open Liberty development mode - OpenLiberty.io
+seo-description: Development mode for Open Liberty allows you to develop applications with any text editor or IDE by providing hot reload and deployment, on demand testing, and debugger support.
+blog_description: "Development mode for Open Liberty allows you to develop applications with any text editor or IDE by providing hot reload and deployment, on demand testing, and debugger support."
+---
+= Open Liberty development mode
+Eric Lau <https://github.com/ericglau>
+
+Open Liberty development mode, or dev mode, allows you to develop your application with any text editor or IDE.  Your code will be automatically compiled and deployed to your running server, making it easy to iterate on your changes. You will be able to run tests on demand or even automatically so that you can get immediate feedback on your changes. You will also be able to attach a debugger at any time to debug your running application.
+
+For a quick demo of what you can do with dev mode, check out the following video:
```

Comment:
Cool. I'm currently trying to get my local build environment working to check that this embedded video displays properly. If you or YK are able to test it properly and it's fine, can you confirm, pls?

Reply:
I haven't tested it in the local build, but the embed format is identical to https://raw.githubusercontent.com/OpenLiberty/blogs/master/publish/2019-09-05-12-factor-microprofile-kubernetes.adoc which shows up in https://openliberty.io/blog/2019/09/05/12-factor-microprofile-kubernetes.html